pages/about.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
 </head>
-<body>
-  
+<body style="background-color:lightcyan;">
+  <nav>
+    <a href="./index.html">Home</a>
+    <a href="./about.html">About</a>
+    <a href="./portfolio.html">Projects</a>
+    <link href="style.css" rel="stylesheet">
+  </nav>
+  <header>
+    <h1>All About Me</h1>
+    <h2>Sorida Lac</h2>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)